src/Config/Config.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+/**
+ * Slim Framework (https://slimframework.com)
+ *
+ * @license https://github.com/slimphp/Slim-Console/blob/0.x/LICENSE.md (MIT License)
+ */
+
+declare(strict_types=1);
+
+namespace Slim\Console\Config;
+
+use ArrayAccess;
+
+use function array_key_exists;
+use function array_merge;
+use function ctype_lower;
+use function is_null;
+use function preg_replace;
+use function strtoupper;
+
+class Config implements ArrayAccess
+{
+    private const ENV_PREFIX = 'SLIM_CONSOLE_';
+
+    /**
+     * @var array<mixed>
+     */
+    private $default = [
+        'bootstrapDir'  => DIRECTORY_SEPARATOR . 'app',
+        'commandsDir'   => DIRECTORY_SEPARATOR . 'Application/Commands',
```

Comment:
Do we need a `DIRECTORY_SEPARATOR` between `Application` and `Commands` too?

Reply:
For the default we do. For user passed input that’ll be up to them!